doc/connectivity/region_extraction.rst

```diff
@@ -24,14 +24,14 @@ Region Extraction for better brain parcellations
 
 .. currentmodule:: nilearn.datasets
 
-Fetching resting state functional datasets
-==========================================
+Fetching movie-watching based functional datasets
+=================================================
 
-We use ADHD resting state functional connectivity datasets of 20 subjects,
-which is already preprocessed and publicly available at
-`<http://fcon_1000.projects.nitrc.org/indi/adhd200/>`_. We use utilities
-:func:`fetch_adhd` implemented in nilearn for automatic fetching of these
-datasets.
+We use naturalistic stimuli based movie watching functional connectivity datasets
+of 20 subjects, which is already preprocessed and publicly available at
+`<https://osf.io/5hju4/files/>`_. We use utilities
+:func:`fetch_development_fmri` implemented in nilearn for automatic fetching of these
+datasets which is downsampled to 4mm.
```

Comment:
Not sure why the 4mm downsampling is especially notable over other processing. Maybe we could add it above? Will try.
```suggestion
dataset.
```